src/Traits/ConvertsEntityToWebsite.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the hyn/multi-tenant package.
+ *
+ * (c) Daniël Klabbers <daniel@klabbers.email>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @see https://laravel-tenancy.com
+ * @see https://github.com/hyn/multi-tenant
+ */
+
+namespace Hyn\Tenancy\Traits;
+
+use Hyn\Tenancy\Contracts\Hostname;
+use Hyn\Tenancy\Contracts\Website;
+
+trait ConvertsEntityToWebsite
```

Comment:
Maybe better name; ConvertHostnameOrWebsiteToWebsite? "Entity" seems so common. It doesnt convert everything.

Reply:
I dislike class names of that length. I know it's not the most perfect trait name, but at least it covers the fact that it tries to retrieve a Website from an Entity.